### PR TITLE
New pack of aws elb

### PR DESCRIPTION
**Pull request by sharonfi99**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9436

## Description
New XSIAM integration of AWS ELB

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS_ELB/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+</~XSIAM>
+# AWS Elastic Load Balancing
+This pack includes Cortex XSIAM content.
+
+## Important Notes
+* The time in this pack is parsed with calculaton of UTC 00:00+.
```
Comment (ShirleyDenkberg) [inline Packs/AWS_ELB/README.md:6]:
```suggestion
Time in this pack is parsed with the calculation of UTC 00:00+.
```

File: Packs/AWS_ELB/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## Collect Events from Vendor
+In order to use the collector, use the [XDRC (XDR Collector)](#xdrc-xdr-collector) option.
+
+### XDRC (XDR Collector)
+To create or configure the Amazon S3 collector, use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Ingest-Network-Flow-Logs-from-Amazon-S3).
+
+You can configure the specific vendor and product for this instance.
+ 
+1. Navigate to **Settings** &rarr; **Data Sources** &rarr **Add Data Source**.
+2. Click on **Amazon S3**.
```
Comment (ShirleyDenkberg) [inline Packs/AWS_ELB/README.md:26]:
```suggestion
2. Click **Amazon S3**.
```

File: Packs/AWS_ELB/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+3. Click **Connect** or **Connect Another Instance**.
+4. Select the **Access Key** or **Assumed Role** filter, due to implementation method of your choice.
```
Comment (ShirleyDenkberg) [inline Packs/AWS_ELB/README.md:28]:
```suggestion
4. Select the **Access Key** or **Assumed Role** filter, according to the implementation method of your choice.
```

**Comment by ShirleyDenkberg** [comment]
@eepstain Doc review completed.

**Review by eepstain — APPROVED** [review]

**Pull request merged**